reference/best_practices/reusable_bundles.rst

```diff
@@ -1,4 +1,98 @@
 Reusable Bundles for Akeneo PIM community
 =========================================
 
-Coming soon!!! :)
+You are an integrator and you want to share your project with the awesome Akeneo PIM community?
+Let's read our advices to make a reusable and documented bundle!
+
+
+What do you want when you create a reusable bundle?
+
+* **Compatibility**: to fit Akeneo PIM features
+* **Maintainability**: to make your code flexible and testable (and tested)
+* **Extensibility**: to allow integrators to use your bundle to fit with their specific needs
+* **Scalability**: to know your application limits and document them
+
+
+Compatibility
+-------------
+
+Your bundle won't be reusable for some customers if you forgot some cases. That is no more an issue if you document compatibilities of your bundle.
+
+Akeneo PIM storage stack comes with MySQL and hybrid MySQL/MongoDB (:doc:`/reference/technical_information/choose_database`).
+Use the ProductQueryBuilder which provides an abstraction layer making both storages work with the same API.
+You can refer to this cookbook for more details and use cases: ":doc:`/cookbook/catalog/product/query`".
+
+Two existing distributions CE/EE. The main project is `Akeneo PIM Community Edition`_ (CE) but
+we also have an `Enterprise Edition`_ (EE) which comes with a couple of extra features dedicated to advanced needs.
```

Comment:
:+1:  I was wrong suggesting "couple of" which is not informal (but might be)  but refers to 2 or 3 things (and not more)